perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -2123,6 +2123,15 @@ public static String prepareGroupNameDscAppsSearchQuerySimilarMatch() {
 		return "(strpos(lower("+Compatibility.convertToAscii("COALESCE(g.name,'') || COALESCE(g.dsc,'')")+"), lower(" + Compatibility.convertToAscii(":searchString") + ")) > 0)";
 	}
 
+	/**
+	 * Returns search query to search by group name or description (similar match) based on databased in use.
+	 *
+	 * @return search query
+	 */
+	public static String prepareApplicationDataSearchQuerySimilarMatch() {
+		return "(strpos(lower("+Compatibility.convertToAscii("COALESCE(d.value,'')")+"), lower(" + Compatibility.convertToAscii(":searchString") + ")) > 0)";
```

Comment:
Compatibility class was used to resolve differences between supported RDBMS. Since we support only Postgres now, you don't need to call methods of this class and you can use directly Postgres-specific expressions in SQL queries. You don't need to change this now, just so you know in the future.